discovery-provider/src/utils/helpers.py

```diff
@@ -19,6 +19,12 @@
 
 from . import multihash
 
+UPDATE_INDEXING_CHECKPOINTS_QUERY = """
```

Comment:
might be better placed in src/queries/query_helpers.py?

Reply:
Sure I'll move it over.

Reply:
in the name of super long files, can we try to avoid query_helpers.py being a huge monolithic thing and instead just put this into its own module? (in an applicable utils / query helpers folder or something like that)